docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,542 @@
+* **Docker**, needed for the `fake data generator for Apache Kafka <https://github.com/aiven/fake-data-producer-for-apache-kafka-docker>`_. Check out the `related installation instructions <https://docs.docker.com/engine/install/>`_.
+* **Slack App and Token**: the data pipeline output is a notifications to a Slack channel, check out the needed steps to `set up a Slack app and retrieve the Slack authentication token <https://github.com/aiven/slack-connector-for-apache-flink>`_.
+* `psql <https://www.postgresql.org/docs/current/app-psql.html>`_ a terminal based tool to interact with PostgreSQL where the threshold data will be stored.
+
+Create the Aiven services
+----------------------------
+
+In this section you'll create the services needed to support the anomaly detection system via the `Aiven Console <https://console.aiven.io/>`_:
+
+* An :doc:`Aiven for Apache Kafka®</docs/products/kafka>` named ``demo-kafka`` for data streaming
+* An :doc:`Aiven for Apache Flink®</docs/products/flink>` named ``demo-flink`` for streaming data transformation
+* An :doc:`Aiven for PostgreSQL®</docs/products/postgresql>` named ``demo-postgresql`` for data storage and query
+
+
+Create an Aiven for Apache Kafka® service
+'''''''''''''''''''''''''''''''''''''''''''''
+
+The :doc:`Aiven for Apache Kafka </docs/products/kafka>` service is responsible to receive the inbound stream of IoT sensor readings. To create the service you need to:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
+
+   .. image:: /images/platform/concepts/console_create_service.png
+      :alt: Aiven Console view for creating a new service
+
+3. Select **Apache Kafka®**.
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `business-4` as service plan.
+
+5. Enter ``demo-kafka`` as name for your service.
+
+6. Click **Create Service** under the summary on the right side of the console
+
+Customise the Aiven for Apache Kafka service
+''''''''''''''''''''''''''''''''''''''''''''
+
+After creating the service, you'll be redirected to the service details page. You can now customise the service to enable the needed components in the *Overview* tab:
+
+1. **Kafka REST API (Karapace)** > **Enable**
+
+   .. Note:: 
+    The **Kafka REST API** enables you to manage Apache Kafka via REST APIs and also to view the data in your Apache Kafka® topics.
+
+2. **Advanced configuration** > **Add configuration option** > ``kafka.auto_create_topics_enable``, switch the setting on and then click **Save advanced configuration**
+
+   .. Note:: 
+    The ``kafka.auto_create_topics_enable`` setting allows you to create new Apache Kafka® topics as you configure your Apache Flink® data tables, so that you do not need to create the topics in advance.
+
+Create an Aiven for PostgreSQL® service
+'''''''''''''''''''''''''''''''''''''''''
```

Comment:
Maybe a sentence about the goal of this PostgreSQL service? It was on the diagram, but some easy to read one sentence can be helpful here too